src/prop/minisat/minisat.cpp

```diff
@@ -170,7 +170,11 @@ void MinisatSatSolver::setupOptions() {
   d_minisat->restart_inc = options().prop.satRestartInc;
 }
 
-ClauseId MinisatSatSolver::addClause(SatClause& clause, bool removable) {
+ClauseId MinisatSatSolver::addClause(SatClause& clause, bool removable)
+{
+  // We disable marking clauses as removable since this has a negative
+  // impact on performance.
+  removable = false;
```

Comment:
I think we should either eliminate the removable flag or call this method with removable=false.

Reply:
Refactored as decided offline.